config/stack/manifests/app.yaml

```diff
@@ -1,5 +1,5 @@
 # Human readable title of application.
-title: Crossplane AWS
+title: Crossplane AWS Provider
 
 overviewShort: The AWS Stack enables resource management with Amazon Web Services.
```

Comment:
want to make other changes to this file to go from `AWS Crossplane Stack` -> `AWS Crossplane Provider`?

note: AWS seems to say AWS Stack instead of AWS Crossplane Stack, which is the format used by other providers (old stacks), fun!